### ISSUE TITLE
'Toggle manual mode' trap

### ISSUE THREAD

**Issue by maartee** [OptiKey/OptiKey#461]
Hi!
This applies if you are fully dependent on eye control only:
opening rightclick menu of optikey, you're one click away from being cut off from interaction with OptiKey and your computer.
If you do a mistake like me and click 'Toggle manual mode' instead of 'Management console', there is no going back using your eyes. (or at least I haven't found it)
Adding of confirmation dialog or removing the option from the menu are my suggestions, if you confirm my findings as legitimate. Thanks!

**Comment by JuliusSweetland** [comment]
@maartee I agree a confirmation on switching *to* manual mode is a good idea.

**Comment by JuliusSweetland** [comment]
@maartee This has been added to master and will be in the next release (2.0.16)
